tests/DwitTech.AccountService.Data.Tests/Repository/UserRepositoryTest.cs

```diff
@@ -381,6 +381,172 @@ public async Task UpdateValidationCode_ShouldUpdateValidationCode_WhenExistingVa
             }
         }
 
+
+        [Fact]
+        public async Task FindPasswordResetToken_Should_Return_True_If_Condition_Is_Satisfied()
+        {
+            // Arrange
+            var options = new DbContextOptionsBuilder<AccountDbContext>()
+                .UseInMemoryDatabase(Guid.NewGuid().ToString())
+                .Options;
+
+            ValidationCode existingValidationCode = new()
+            {
+                Id = 1,
+                UserId = 1,
+                Code = "9fPn1CFhKXoFMa72dmSh",
+                CodeType = CodeType.ResetToken
+            };
+           
+            using (var accountDbContext = new AccountDbContext(options))
+            {
+
+                accountDbContext.ValidationCodes.Add(existingValidationCode);
+                await accountDbContext.SaveChangesAsync();
+
+                var userRepository = new UserRepository(accountDbContext);
+
+               //Act
+                var token = "9fPn1CFhKXoFMa72dmSh";
+                var result = userRepository.FindPasswordResetToken(token);
+
+                Assert.True(result);
+            }
+        }
+
+        [Fact]
+        public async Task GetUserIdByPasswordResetToken_Should_Pass_If_UserId_Are_Same()
+        {
+            // Arrange
+            var options = new DbContextOptionsBuilder<AccountDbContext>()
+                .UseInMemoryDatabase(Guid.NewGuid().ToString())
+                .Options;
+
+            ValidationCode existingValidationCode = new()
+            {
+                Id = 1,
+                UserId = 1,
+                Code = "9fPn1CFhKXoFMa72dmSh",
+                CodeType = CodeType.ResetToken
+            };
+            
+            using (var accountDbContext = new AccountDbContext(options))
+            {
+
+                accountDbContext.ValidationCodes.Add(existingValidationCode);
+                await accountDbContext.SaveChangesAsync();
+
+                var userRepository = new UserRepository(accountDbContext);
+
+                //Act
+                var token = "9fPn1CFhKXoFMa72dmSh";
+                var result = await userRepository.GetUserIdByPasswordResetToken(token);
+
+                Assert.Equal(result,existingValidationCode.UserId);
+            }
+        }
+
+        [Fact]
+        public async Task GetUserIdByPasswordResetToken_Should_Pass_If_UserId_Are_Not_Same()
```

Comment:
Please correct the name of the test. It should not pass

Reply:
I feel since  assert is testing for not equal and the test satisfies this condition then the test passes, so the name seem okay except if you think otherwise